LEGO1/lego/legoomni/src/common/legoanimationmanager.cpp

```diff
@@ -1,13 +1,22 @@
 #include "legoanimationmanager.h"
 
+#include "legocharactermanager.h"
 #include "legogamestate.h"
 #include "legoomni.h"
 #include "misc.h"
 #include "mxutilities.h"
+#include "roi/legoroi.h"
 
 #include <io.h>
 
 DECOMP_SIZE_ASSERT(LegoAnimationManager, 0x500)
+DECOMP_SIZE_ASSERT(Character, 0x18)
+DECOMP_SIZE_ASSERT(Vehicle, 0x8)
+DECOMP_SIZE_ASSERT(Unknown0x3c, 0x18)
+
+// GLOBAL: LEGO1 0x100f6d10
+Vehicle g_vehicles[] = {"bikebd", 0,        FALSE, "bikepg", 0,        FALSE, "bikerd", 0,       FALSE, "bikesy", 0,
```

Comment:
The global doesn't check out:

![image](https://github.com/isledecomp/isle/assets/1135351/43189466-22fa-47d3-86f4-1ed0c8bb0f22)